portfolio/src/main/webapp/script.js

```diff
@@ -36,3 +36,30 @@ function addRandomFact() {
   funFactContainer.innerText = funFact;
   funFactContainer.style.backgroundColor = "whitesmoke";
 }
+
+function getComments() {
+  const selectElement = document.getElementById("limit");
+  const commentLimit = selectElement.options[selectElement.selectedIndex].value;
+  fetch("/data?limit=" + commentLimit).then(response => response.json()).then((comments) => {
+    clearComments();
+    comments.forEach(addComment);
+  });
+}
+
+function addComment(comment) {
+  const commentContainer = document.getElementById('comment-container');
+  const node = document.createElement('li');
+  node.setAttribute("id", "comment");
+  const textNode = document.createTextNode(comment.text);
+  node.appendChild(textNode);
+  commentContainer.appendChild(node);
+}
+
+function clearComments() {
+  const commentContainer = document.getElementById('comment-container');
+  commentContainer.innerHTML = '';
+}
+
+function deleteComments() {
+  fetch("/delete-data", {method: 'POST'}).then(() => getComments());
```

Comment:
Let's use single quote instead of double quote for string literals.

Reference:
https://google.github.io/styleguide/jsguide.html#features-strings-use-single-quotes

Reply:
Done.